ruby/ql/test/library-tests/frameworks/Eval.rb

```diff
@@ -1,24 +0,0 @@
-# Uses of eval and send
```

Comment:
This file is supposed to be used by https://github.com/github/codeql/commit/95e50cedade0c67c0a1207c0a21a8ee23b3d0dfb#diff-fb73d3e403dc11874b7ee389cdffdc64a1fcab64f7369d5778adcbbb7c55cbf5